Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/) for Kodi add-ons: services/service.subsmangler.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-services/-/service-subsmangler/horizontal-auto.svg)

gade01 reviewed (approved):
Looks good to me.

- Corrections are done
- New translations

weblate committed:
Added translation using Weblate (Persian (Iran) (fa_ir))

Added translation using Weblate (Tamil (India) (ta_in))

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 14.2% (15 of 105 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 18.0% (19 of 105 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 20.9% (22 of 105 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 20.9% (22 of 105 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 24.7% (26 of 105 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 62.8% (66 of 105 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 58.0% (61 of 105 strings)

Translated using Weblate (German (de_de))

Currently translated at 58.0% (61 of 105 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 62.8% (66 of 105 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (105 of 105 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 62.8% (66 of 105 strings)

Translated using Weblate (German (de_de))

Currently translated at 58.0% (61 of 105 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 62.8% (66 of 105 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 52.3% (55 of 105 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 5.7% (6 of 105 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Edson Armando <edsonarmando78@outlook.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/sk_sk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-services/service-subsmangler/sq_al/
Translation: Kodi add-ons: services/service.subsmangler
